### DISCUSSION TITLE
Custom function to translate literals from one language to an other

### DISCUSSION THREAD

**Discussion by JervenBolleman** [eclipse-rdf4j/rdf4j#4640] [Show and tell]
Hi All,

I just wrote a small custom function that uses libretranslate to translate a literal from one language to an other.
Would be very happy if people give suggestions to improve it.

https://github.com/JervenBolleman/libretranslate-rdf4j

Regards,
Jerven

**Comment by hmottestad** [comment]
I think that's pretty cool.

We already have a section on https://rdf4j.org about third party databases, we could have a section on third party functions. What do you think?
